fbsimctl/fbsimctl.xcodeproj/project.pbxproj

```diff
@@ -610,13 +610,15 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = AA7D2F5F1D33844000C7C6CF /* fbsimctl.xcconfig */;
 			buildSettings = {
+				LD_RUNPATH_SEARCH_PATHS = "@executable_path/../Frameworks $(inherited)";
```

Comment:
You should be able to put this in the `fbsimctl.xcconfig` instead of the Project itself.